api/v1alpha1/database_webhook.go

```diff
@@ -42,65 +44,78 @@ func (r *Database) SetupWebhookWithManager(mgr ctrl.Manager) error {
 
 var _ webhook.Defaulter = &Database{}
 
-func instanceSpecDefaulterForCreate(r *Database) {
+func instanceSpecDefaulterForCreate(instance *Instance) {
```

Comment:
Can we rename this to databaseInstance ?

Reply:
This is picked from "r.Spec.Instance" field in the spec, so for consistency purpose, "Instance" is a better option.